### PR TITLE
Home Screen

### PR DESCRIPTION
**Pull request by horsemaker**
- Added Main Layout
- Added Home Screen
- Added Quotes Data
- Added links in Header

recording

### CODE REVIEW

File: src/frontend/layouts/Main/Main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { Route, Routes } from "react-router-dom";
+import { HomeScreen } from "../../screens";
+import "./Main.css";
+
+export const Main = () => {
+  return (
+    <Routes>
+      <Route path="/" element={<HomeScreen />} />
+      <Route path="/signup" element={<></>} />
+      <Route path="/signip" element={<></>} />
```
Comment (MarkVed17) [inline src/frontend/layouts/Main/Main.js]: signin?

Comment (horsemaker) [reply src/frontend/layouts/Main/Main.js]: Yes. Thank you!

File: src/frontend/layouts/Header/Header.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { NavLink, useLocation } from "react-router-dom";
+import { NavLink, useLocation, useNavigate } from "react-router-dom";
```
Comment (MarkVed17) [inline src/frontend/layouts/Header/Header.js:2]: NavLink isn't used anywhere ig

**Review by MarkVed17 — APPROVED** [review]
Great work there with the UI @horsemaker 
I really liked the inclusion of quotes. Looks sleek!

File: src/frontend/layouts/Header/Header.js (hunk 2 above)
Comment (horsemaker) [reply src/frontend/layouts/Header/Header.js:2]: It is, it is.

File: src/frontend/screens/HomeScreen/HomeScreen.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
+.home-screen {
+  margin-left: 1rem;
+  margin-right: 1rem;
```
Comment (t007rushi) [inline src/frontend/screens/HomeScreen/HomeScreen.css:3]: can make it one-liner

File: src/frontend/screens/HomeScreen/HomeScreen.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+  return (
+    <div className="home-screen main">
+      <section className="home-screen-details">
+        <h1 className="home-screen-title">Meet your modern note taking app</h1>
+        <figure className="home-screen-quote">
+          <blockquote>
+            {quotes[Math.floor(Math.random() * quotes.length)]}
+          </blockquote>
+          <figcaption className="home-screen-quote-caption">
```
Comment (t007rushi) [inline src/frontend/screens/HomeScreen/HomeScreen.js:20]: Liked the quote change acc. to theme Nicely done

**Review by t007rushi — APPROVED** [review]

File: src/frontend/screens/HomeScreen/HomeScreen.css (hunk 3 above)
Comment (horsemaker) [reply src/frontend/screens/HomeScreen/HomeScreen.css:3]: I will look into it. 👍

**Pull request merged**